authentication/src/main/java/io/camunda/authentication/CamundaUserDetailsService.java

```diff
@@ -63,12 +69,18 @@ public UserDetails loadUserByUsername(final String username) throws UsernameNotF
             .flatMap(Set::stream)
             .collect(Collectors.toList());
 
+    final var tenants =
+        tenantServices.getTenantsByMemberKey(storedUser.userKey()).stream()
```

Comment:
❓ should we consider user's groups tenants ?

Reply:
I'm not sure about the semantics here. Do we have groups (and roles) that belong to the tenant or does membership in a group (or role) grant access to the tenant?

Reply:
![image](https://github.com/user-attachments/assets/8c364bff-743f-4820-83ce-692183e9d4aa)
from https://camunda.slack.com/archives/C06UYJMMETZ/p1733759374586799?thread_ts=1733756519.685409&cid=C06UYJMMETZ
I guess the group implementation is still in progress